### ISSUE TITLE
Issue: Plugin Load Error

### ISSUE THREAD

**Issue by bntzio** [klauscfhq/hyper-pokemon#7]
I'm getting this Hyper error when following the installation steps:

![screenshot](https://cl.ly/l3IG/Screen%20Shot%202017-06-19%20at%2011.41.50%20AM.png)

Also, in the dev console I get this:

![screenshot2](https://cl.ly/l8ci/Screen%20Shot%202017-06-19%20at%2011.45.01%20AM.png)

Here's my hyper.js config file:

    module.exports = {
    config: { 
    // default font size in pixels for all tabs
    fontSize: 12,

    // font family with optional fallbacks
    fontFamily: 'Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',

    // terminal cursor background color and opacity (hex, rgb, hsl, hsv, hwb or cmyk)
    cursorColor: 'rgba(92,230,205,0.8)',

    // `BEAM` for |, `UNDERLINE` for _, `BLOCK` for █
    cursorShape: 'BLOCK',

    // color of the text
    foregroundColor: '#5ce6cd',

    // terminal background color
    backgroundColor: '#000',

    // border color (window, tabs)
    borderColor: '#000',

    // custom css to embed in the main window
    css: '',

    // custom css to embed in the terminal window
    termCSS: '',

    // set to `true` (without backticks) if you're using a Linux setup that doesn't show native menus
    // default: `false` on Linux, `true` on Windows (ignored on macOS)
    showHamburgerMenu: '',

    // set to `false` if you want to hide the minimize, maximize and close buttons
    // additionally, set to `'left'` if you want them on the left, like in Ubuntu
    // default: `true` on windows and Linux (ignored on macOS)
    showWindowControls: '',

    // custom padding (css format, i.e.: `top right bottom left`)
    padding: '12px 14px',

    // the full list. if you're going to provide the full color palette,
    // including the 6 x 6 color cubes and the grayscale map, just provide
    // an array here instead of a color map object
    colors: {
      black: '#000000',
      red: '#ff0000',
      green: '#33ff00',
      yellow: '#ffff00',
      blue: '#0066ff',
      magenta: '#cc00ff',
      cyan: '#00ffff',
      white: '#d0d0d0',
      lightBlack: '#808080',
      lightRed: '#ff0000',
      lightGreen: '#33ff00',
      lightYellow: '#ffff00',
      lightBlue: '#0066ff',
      lightMagenta: '#cc00ff',
      lightCyan: '#00ffff',
      lightWhite: '#ffffff'
    },

    // hyper-pokemon config
    pokemon: 'random',
    pokemonSyntax: 'dark',
    unibody: 'true',

    // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
    // if left empty, your system's login shell will be used by default
    // make sure to use a full path if the binary name doesn't work
    // (e.g `C:\\Windows\\System32\\bash.exe` instad of just `bash.exe`)
    // if you're using powershell, make sure to remove the `--login` below
    shell: 'zsh',

    // for setting shell arguments (i.e. for using interactive shellArgs: ['-i'])
    // by default ['--login'] will be used
    shellArgs: ['--login'],

    // for environment variables
    env: {},

    // set to false for no bell
    bell: 'SOUND',

    // if true, selected text will automatically be copied to the clipboard
    copyOnSelect: false,

    // if true, on right click selected text will be copied or pasted if no
    // selection is present (true by default on Windows)
    // quickEdit: true

    // URL to custom bell
    // bellSoundURL: 'http://example.com/bell.mp3',

    // for advanced config flags please refer to https://hyper.is/#cfg
    
    // hyperterm-overlay settings
    // overlay: {
    //   alwaysOnTop: false,
    //   animate: false,
    //   hasShadow: false,
    //   hideDock: false,
    //   hideOnBlur: false,
    //   hotkeys: ['Cmd+Space'],
    //   position: 'top',
    //   primaryDisplay: false,
    //   resizable: true,
    //   startAlone: false,
    //   startup: false,
    //   size: 0.5,
    //   tray: true,
    //   unique: false
    // },
    },

    // a list of plugins to fetch and install from npm
    // format: [@org/]project[#version]
    // examples:
    //   `hyperpower`
    //   `@company/project`
    //   `project#1.0.1`
    plugins: [// 'hyper-electron-highlighter',
    // 'hyperterm-hipster',
    // 'hyperambient',
    // 'hyperline',
    // 'htyt',
    // 'hyper-transparent-bg',
    // 'hyperterm-tabby',
    // 'hyper-always-on-top',
    // 'hyper-pane',
    // 'hyperterm-overlay',
    // 'hyperborder',
    'hyperminimal', 'hyperlinks', 'hypercwd', 'hyperterm-alternatescroll', 'hyper-snazzy', 'hyperterm-tab-numbers', 'hyper-tabs-enhanced', 'hyper-dark-scrollbar', "hyper-pokemon"],
    // in development, you can create a directory under
    // `~/.hyper_plugins/local/` and include it here
    // to load it and avoid it being `npm install`ed
    localPlugins: []
    };

**OS** MacOS Sierra v10.12.5

What could be wrong? Thanks! 😄

**Comment by bntzio** [comment]
Oh, never mind! I got it working now, I just closed Hyper and opened it again, the error is gone now, fantastic themes btw! 👍

**Comment by klauscfhq** [comment]
@bntzio No worries at all! Thank you tons for dropping by and for your kind words! You are always welcome! : )

**Comment by codeofsumit** [comment]
same error but restart doesn't help :(
I get it with every restart

**Comment by fromthemills** [comment]
I solved this by setting the `pokemon`, `pokemonSyntax`  & `unibody` props in the `~/.hyper.js` file. See the [Usage](https://github.com/hyper-pokemon/hyper-pokemon#usage) section

**Comment by codeofsumit** [comment]
my problem was related to this:
https://github.com/zeit/hyper/issues/191

so nothing with hyper-pokemon I guess

**Comment by bnb** [comment]
Also having this - funnily enough, I filed https://github.com/zeit/hyper/issues/191 and don't know how to fix it 😅

**Comment by klaudiosinani** [comment]
Indeed, it's something that we have also noticed. Some reliable & possible solutions atm are:

- Trying to fully reload Hyper with <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>R</kbd>
- Manually installing `hyper-pokemon` from scratch. Here is a quick guide on how to do it: 

   ```bash
   # clone the repo inside `~/.hyper_plugins/node_modules/hyper-pokemon`
   $ git clone https://github.com/hyper-pokemon/hyper-pokemon.git
   # navigate to the project directory
   $ cd hyper-pokemon
   # install the dependencies
   $ npm install
   ```
-  Downgrading to `npm@4` (if you are not already on it) using `npm install -g npm@latest-4`

**Edit:** More new successfully proven solutions have been proposed and implemented: 
- by @awaisabir [here](https://github.com/hyper-pokemon/hyper-pokemon/issues/7#issuecomment-310581788)
- by @beilunyang [here](https://github.com/hyper-pokemon/hyper-pokemon/issues/7#issuecomment-319846563) 
- by @sdkcarlos [here](https://github.com/hyper-pokemon/hyper-pokemon/issues/18#issuecomment-321047755)

~These three ways are the only ones, to our experience, sometimes solve the issue on MacOS Sierra 10.12.5, but they do not completely prevent it from re-appearing.~ We will keep testing them, though it seems more like a `hyper` related issue than a `hyper-pokemon` one, since it also appears on other `hyper` plugins too. If anything new comes up, we will let you all know as soon as possible. Thank you all for dropping by and sharing your thoughts : ) 

*We will keep this issue as **Open** until a ~more~ fully credible solution is established*

**Comment by klauscfhq** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Plugin Loading Error ](https://trello.com/c/mF1MDlur/36-plugin-loading-error)

**Comment by awaisabir** [comment]
I was getting the same error, however I fixed it by simply using `yarn` instead of `npm`. Simply `cd C:\Users\%USERNAME%\.hyper_plugins` and run `yarn install`. That should do the trick, In my opinion it is a better option than downgrading your `npm` version.

Cheers! :beers:

**Comment by georgebax** [comment]
Wow, this _does_ seem like a pretty good fix! We will be testing the theme on various OS configurations and will close this issue if the error doesn't find its way back... :robot:

**Comment by awaisabir** [comment]
@georgebax The problem is that when you use `npm` to install the dependencies it creates the `package-lock.json` file which somehow makes hyper act in a weird way, and reject the plugin. Hyper is defaulting to your `npm` version and installing the plugins. This is very troublesome. Hyper should default to `yarn`. This issue is a problem with hyper itself, and not with your plugin...

**Comment by georgebax** [comment]
Yes we have encountered numerous problems with `npm` ourselves, it can be really buggy sometimes, especially along with `hpm-cli`.  Anyway, should you encounter any other issues / have any tips, feel free to contact us!

**Comment by beilunyang** [comment]
i don't use hpm install it. try these cmd:
```
cd ~/.hyper_plugins 
vim package.json
add "hyper-pokemon": "latest" in the dependencies
sudo npm install
```
finally, resolved the problem successfully.

**Comment by klaudiosinani** [comment]
[Issue](https://github.com/zeit/hyper/issues/191) considered solved since [`Hyper@1.4.0`](https://github.com/zeit/hyper/releases/latest)
